website/docs/guides/guides-upgrade.md

```diff
@@ -45,77 +45,124 @@ If you executed the previous step already on your server, then obviously nothing
 Then restore the original installation from your backup.
 Either by extracting (`gunzip ls_backup.tar.gz && tar xvf ls_backup.tar`) or moving back the copied folder (`mv <ls_folder>.backup <ls_folder>`).
 Or: Re-uploading your backup, if that was the way you went.
 
-### Unlock installation
+## Pitfalls and Known Bugs
 
-As the backup was taken at a point where the installation was locked, it may still be required to unlock it
-This would be done by reverting whatever was done to facilitate the original lock
+* Please ensure that export and import of Database images use the same character encoding. Using the same client on the same system should ensure this, but be cautious if dump and import are done on different systems/clients.
+* The git `master` branch is not usable without pulling in additional resources via composer. You must do this first to obtain a runnable installation!
 
-## Pitfalls and Known Bugs
+## Upgrade from LanSuite v4.2 to v5.0
+
+### Configuration: `database.sqlmode`
+
+Add the following line to your configuration at `/inc/base/config.php`:
+
+```
+[...]
+[database]
+sqlmode = "NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
+[...]
+```
+
+### Configuration: `database.charset`
+
+Add the following line to your configuration at `/inc/base/config.php`:
+
+```
+[...]
+[database]
+charset = "utf8mb4"
+[...]
+```
+
+### Fonts in `ext_inc/pdf_fonts`
+
+Delete the following files
+
+* `ext_inc/pdf_fonts/courier.php`
+* `ext_inc/pdf_fonts/helvetica.php`
+* `ext_inc/pdf_fonts/helveticab.php`
+* `ext_inc/pdf_fonts/helveticabi.php`
+* `ext_inc/pdf_fonts/helveticai.php`
+* `ext_inc/pdf_fonts/symbol.php`
+* `ext_inc/pdf_fonts/times.php`
+* `ext_inc/pdf_fonts/timesb.php`
+* `ext_inc/pdf_fonts/timesbi.php`
+* `ext_inc/pdf_fonts/timesi.php`
+* `ext_inc/pdf_fonts/zapfdingbats.php`
+
+Delete the following folder
+
+* `ext_inc/pdf_fonts/makefont`
+
+Add the following files from the release package to the `ext_inc/pdf_fonts` folder
+
+* `ext_inc/pdf_fonts/.gitkeep`
+
+### Converting custom fonts
+
+If you have added your own custom fonts into `ext_inc/pdf_fonts`, you need to convert them into the new fpdf format:
 
-* Please ensure that export and import of Database images use the same character encoding. Using the same client on the same system should ensure this, but be cautious if dump and import are done on different systems/clients
-* MySQL 5.7 changes default behavior for GROUP BY clauses. This may lead to unexpected errors in some cases. See [#117](https://github.com/lansuite/lansuite/issues/117) for details
-* The master branch is not usable without pulling in additional resources via composer. You must do this first to obtain a runnable installation!
-* You have to run a database table upgrade as an extension of the IP field is required to store logging information
```

Comment:
Do we need to provide a conversion logic for existing DB entries in IPv4?

Reply:
This part is removed. As far a I know this is not needed anymore. I have updated my system already and never did such a conversion

Reply:
Had a look. It appears that old column values are dropped with the related column change.
So there is nothing to convert once the DB update is done.
Not sure if feature or bug :/

Reply:
Storing the ipv4 addresses without a proper reason is not gdpr confirm either. In this case, I don't think we need to fix this.

On top of this, people have a database backup. If they really need the IPv4 addresses, there is a possibility to add those.

Reply:
I would suggest to add a note that this happens and leave it with that.
log is (supposed to be) cleaned up regularly via cron2, so these should disappear in reasonable time anyways.

Ad GDPR:
The proper reason is legimimate interest and is actually fine as long as it is disclosed transparently (where we fail at the moment). 
(Apparently there are also some court cases in DE/AU where dynamic IPs were not considered personal information, but as so often it is not clear-cut)

Reply:
Done in https://github.com/lansuite/lansuite/pull/794/commits/929e27a9c2898c8c2fbe7ea9ad769069dbffbd91